icon-registry/icons-static/technology-accessibility-disabled_x48.svg

```diff
@@ -0,0 +1,7 @@
+<svg  viewBox="0 0 48 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@emilyrohrbough @emilmilanov This icon appears to be `48x49` instead of `48x48`. Probably worth fixing that in case it ever needs to be centered properly once it's used.

Reply:
In Figma icon is 48x48. I'm not sure how we ended up with 48x49.

I'm getting the code below when I export svg from Figma. cc @emilyrohrbough 

`<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect x="5" y="4.99927" width="38" height="38" rx="19" fill="#D0D2E0" stroke="#1B1E2E" stroke-width="2"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M24 19C25.3807 19 26.5 17.8807 26.5 16.5C26.5 15.1193 25.3807 14 24 14C22.6193 14 21.5 15.1193 21.5 16.5C21.5 17.8807 22.6193 19 24 19ZM17.2425 19.0299C16.7067 18.8959 16.1638 19.2217 16.0298 19.7575C15.8959 20.2933 16.2217 20.8362 16.7575 20.9701L23 22.5308V26.6667L17.95 33.4C17.6186 33.8418 17.7082 34.4686 18.15 34.8C18.5918 35.1314 19.2186 35.0418 19.55 34.6L24 28.6667L28.45 34.6C28.7814 35.0418 29.4082 35.1314 29.85 34.8C30.2918 34.4686 30.3814 33.8418 30.05 33.4L25 26.6667V22.5308L31.2425 20.9701C31.7783 20.8362 32.1041 20.2933 31.9701 19.7575C31.8362 19.2217 31.2933 18.8959 30.7575 19.0299L24 20.7192L17.2425 19.0299Z" fill="#1B1E2E"/>
<circle cx="37" cy="37" r="8" fill="white"/>
<path d="M43 37.001C43 33.6873 40.3137 31.001 37 31.001C33.6863 31.001 31 33.6873 31 37.001C31 40.3147 33.6863 43.001 37 43.001C40.3137 43.001 43 40.3147 43 37.001Z" fill="#E1E3ED"/>
<path d="M32.7412 41.2274L41.2656 32.7814M43 37.001C43 33.6873 40.3137 31.001 37 31.001C33.6863 31.001 31 33.6873 31 37.001C31 40.3147 33.6863 43.001 37 43.001C40.3137 43.001 43 40.3147 43 37.001Z" stroke="#9095AD" stroke-width="2" stroke-linecap="round"/>
</svg>`

Reply:
I'm not sure, I exported it as well 😅 I'll try it again

Reply:
Weird! Just exported again and have 48x49